src/Optix.cpp

```diff
@@ -181,22 +177,21 @@ Optix::~Optix()
 
 void Optix::initializeStaticOptixStructures()
 {
-	OptixModuleCompileOptions moduleCompileOptions = {
-		.maxRegisterCount = 100,
+	OptixModuleCompileOptions moduleCompileOptions = { .maxRegisterCount = 100,
 #ifdef NDEBUG
-		.optLevel = OPTIX_COMPILE_OPTIMIZATION_LEVEL_2,
-		.debugLevel = OPTIX_COMPILE_DEBUG_LEVEL_NONE
+		                                               .optLevel = OPTIX_COMPILE_OPTIMIZATION_LEVEL_2,
+		                                               .debugLevel = OPTIX_COMPILE_DEBUG_LEVEL_NONE
 #else
-		.optLevel = OPTIX_COMPILE_OPTIMIZATION_LEVEL_0,
-		.debugLevel = OPTIX_COMPILE_DEBUG_LEVEL_FULL
+		                                               .optLevel = OPTIX_COMPILE_OPTIMIZATION_LEVEL_0,
+		                                               .debugLevel = OPTIX_COMPILE_DEBUG_LEVEL_FULL
 #endif
```

Comment:
Because the first `.fieldName = value` is in the same line, this is weirdly formatted. If possible, I'd like clang-format to fix this by putting the linebreak after `{` and before the first `.fieldName = value`. If not, just add a linebreak here manualy.